src/function/open-referral-function/src/FamilyHubs.OpenReferral.Function/Functions/TriggerPullServicesWebhook.cs

```diff
@@ -0,0 +1,67 @@
+using System.Net;
+using FamilyHubs.OpenReferral.Function.ClientServices;
+using FamilyHubs.OpenReferral.Function.Entities;
+using FamilyHubs.OpenReferral.Function.Repository;
+using FamilyHubs.ServiceDirectory.Data.Entities.Staging;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Extensions.Logging;
+using Microsoft.Azure.Functions.Worker.Http;
+using Newtonsoft.Json.Linq;
+
+namespace FamilyHubs.OpenReferral.Function.Functions;
+
+public class TriggerPullServicesWebhook(
+    ILogger<TriggerPullServicesWebhook> logger,
+    IHsdaApiService hsdaApiService,
+    IFunctionDbContext functionDbContext)
+{
+    [Function("TriggerPullServicesWebhook")]
+    public async Task<HttpResponseData> Run([HttpTrigger(AuthorizationLevel.Function, "POST")] HttpRequestData req)
+    {
+        logger.LogInformation("[ApiReceiver] HTTP Trigger Function Started");
+
+        (HttpStatusCode httpStatusCode, JArray? services) = await hsdaApiService.GetServices();
+
+        if (httpStatusCode != HttpStatusCode.OK) return req.CreateResponse(httpStatusCode);
+
+        List<ServiceJson> servicesById = await hsdaApiService.GetServicesById(services!);
+
+        if (servicesById.Count == 0)
+        {
+            logger.LogInformation("Getting the services by ID returned no results!");
+            return req.CreateResponse(HttpStatusCode.NotFound);
+        }
+
+        try
+        {
+            await UpdateDatabase(servicesById);
+        }
+        catch (Exception e)
+        {
+            logger.LogError("{exception}", e.Message);
+            return req.CreateResponse(HttpStatusCode.InternalServerError);
+        }
+
+        return req.CreateResponse(HttpStatusCode.OK);
+    }
+
+    private async Task UpdateDatabase(List<ServiceJson> serviceJsonList)
+    {
+        logger.LogInformation("Truncating database before inserting services..");
```

Comment:
nitpick(big time): .. -> ...?

Reply:
Just got rid of em: https://github.com/DFE-Digital/fh-services/pull/63/commits/7c558752020a818b377f73482fa808152a8fc6fe